### PR TITLE
fix: fix bug of release chart

### PR DESCRIPTION
**Pull request by codeflitting**
**What this PR does / why we need it**:

* default_resource_quota： Without default_resource_quota can't start pod
* native-cyclone-server：This causes the pv name to be created too long

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: release/workflow-controller.yaml [hunk 1]
```diff
@@ -42,6 +42,12 @@ _config:
             "limits": {
               "max_workflowruns": 50
             },
+            "default_resource_quota": {
```
Comment (supereagle) [inline release/workflow-controller.yaml]: Missing limit? Please refer to https://github.com/caicloud/cyclone/blob/develop/manifests/cyclone.yaml#L29-L37

Comment (codeflitting) [reply release/workflow-controller.yaml]: need limit？

Comment (supereagle) [reply release/workflow-controller.yaml]: yes

File: release/workflow-controller.yaml [hunk 2]
```diff
@@ -42,6 +42,12 @@ _config:
...
+              "requests": {
+                "cpu": "100m",
+                "memory": "128Mi"
+              }
+            },
             "pvc": "native-cyclone-server-server-v1-0-cyclone-data",
             "cyclone_server_addr": "native-cyclone-server.default:7099"
```
Comment (supereagle) [inline release/workflow-controller.yaml]: Please change the port to `7089`, I change this by mistake yesterday.

Comment (codeflitting) [reply release/workflow-controller.yaml]: done

File: release/workflow-controller.yaml (hunk 1 above)
Comment (codeflitting) [reply release/workflow-controller.yaml]: done

**Pull request merged**